src/clients/authorization/AuthorizationEvaluate.tsx

```diff
@@ -0,0 +1,325 @@
+import { HelpItem } from "../../components/help-enabler/HelpItem";
+import type { ClientForm } from "../ClientDetails";
+import { FormPanel } from "../../components/scroll-form/FormPanel";
+import type UserRepresentation from "@keycloak/keycloak-admin-client/lib/defs/userRepresentation";
+import type RoleRepresentation from "@keycloak/keycloak-admin-client/lib/defs/roleRepresentation";
+import { useAdminClient, useFetch } from "../../context/auth/AdminClient";
+import type ResourceEvaluation from "@keycloak/keycloak-admin-client/lib/defs/resourceEvaluation";
+import { useRealm } from "../../context/realm-context/RealmContext";
+import { AttributeInput } from "../../components/attribute-input/AttributeInput";
+import { defaultContextAttributes } from "../utils";
+import type ResourceRepresentation from "@keycloak/keycloak-admin-client/lib/defs/resourceRepresentation";
+import { useParams } from "react-router-dom";
+
+export type AttributeType = {
+  key: string;
+  name: string;
+  custom?: boolean;
+  values?: {
+    [key: string]: string;
+  }[];
+};
+
+type ClientSettingsProps = {
+  clients: ClientRepresentation[];
+  clientName?: string;
+  save: () => void;
+  reset: () => void;
+  users: UserRepresentation[];
+  clientRoles: RoleRepresentation[];
+};
+
+export const AuthorizationEvaluate = ({
+  clients,
+  clientRoles,
+  clientName,
+  users,
+  reset,
+}: ClientSettingsProps) => {
+  const { control } = useFormContext<ClientForm>();
+  const form = useForm<ResourceEvaluation>({ mode: "onChange" });
+  const { t } = useTranslation("clients");
+  const adminClient = useAdminClient();
+  const realm = useRealm();
+  const { clientId } = useParams<{ clientId: string }>();
+
+  const [clientsDropdownOpen, setClientsDropdownOpen] = useState(false);
+  const [userDropdownOpen, setUserDropdownOpen] = useState(false);
+  const [roleDropdownOpen, setRoleDropdownOpen] = useState(false);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [resources, setResources] = useState<ResourceRepresentation[]>([]);
+
+  useFetch(
+    () => {
+      return adminClient.clients.listResources({ id: clientId });
+    },
+    (resources) => setResources(resources),
+    []
+  );
+
+  const evaluate = () => {
+    const formValues = form.getValues();
+    const resEval: ResourceEvaluation = {
+      roleIds: formValues.roleIds ?? [],
+      userId: users.find((user) => user.username === formValues.userId)?.id!,
+      entitlements: false,
+      context: formValues.context,
+      clientId: clients.find(
+        (client) => client.clientId === formValues.clientId
+      )?.id!,
+    };
+    return adminClient.clients.evaluateResource(
+      { id: clientId!, realm: realm.realm },
+      resEval
+    );
+  };
+
+  return (
+    <PageSection>
+      <FormPanel
+        className="kc-identity-information"
+        title={t("identityInformation")}
+      >
+        <FormAccess isHorizontal role="manage-clients">
```

Comment:
use form submit so it only calls your `evaluate` function when the form is valid
```suggestion
        <FormAccess isHorizontal role="manage-clients" onSubmit={form.handleSubmit(evaluate)}>
```